src/main/java/com/hm/petmaster/listener/PlayerTameListener.java

```diff
@@ -0,0 +1,39 @@
+package com.hm.petmaster.listener;
+
+import com.hm.petmaster.PetMaster;
+import java.io.File;
+import org.bukkit.DyeColor;
+import org.bukkit.entity.Cat;
+import org.bukkit.entity.Wolf;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.EventPriority;
+import org.bukkit.event.Listener;
+import org.bukkit.event.entity.EntityTameEvent;
+
+public class PlayerTameListener implements Listener {
+
+	private final PetMaster plugin;
+
+	public PlayerTameListener(PetMaster plugin) {
+		this.plugin = plugin;
+	}
+
+	@EventHandler(priority = EventPriority.NORMAL, ignoreCancelled = true)
+	public void onPlayerTamePet(EntityTameEvent event) {
+		switch (event.getEntity().getType()) {
+			case CAT: {
```

Comment:
`CAT` was introduced in Minecraft version 1.14. The plugin is still compatible with any version from 1.17.9 to 1.13.2, so you should probably be a bit careful here, as this will throw a runtime exception on older server versions:
* either switch on `event.getEntity().getType().name()` and match with strings instead of the enum values. On older server versions, the string `"CAT"` will never match, and the `((Cat) event.getEntity())` code will never be executed, so no errors there either.
* either do an explicit server version check, there's an example here: https://github.com/PyvesB/PetMaster/blob/0c3037d223d14f144c95dcfad67f1970bcd451c4/src/main/java/com/hm/petmaster/listener/PlayerInteractListener.java#L64

Reply:
Good suggestion, i completely missed checking for backwards compatibility.